Src/GoogleApis.Auth/OAuth2/ITokenAccess.cs

```diff
@@ -22,19 +22,24 @@
 namespace Google.Apis.Auth.OAuth2
 {
     /// <summary>
-    /// Allows direct control over access token and its retrieval.
+    /// Allows direct retrieval of access tokens to authenticate requests.
     /// This is necessary for workflows where you don't want to use 
     /// <see cref="T:Google.Apis.Services.BaseClientService"/> to access the API.
     /// (e.g. gRPC that implemenents the entire HTTP2 stack internally).
     /// </summary>
     public interface ITokenAccess
     {
-        /// <summary>Gets the token response which contains the access token.</summary>
-        TokenResponse Token { get; }
-
-        /// <summary>Requests a new token.</summary>
-        /// <param name="taskCancellationToken">Cancellation token to cancel operation.</param>
-        /// <returns><c>true</c> if a new token was received successfully.</returns>
-        Task<bool> RequestAccessTokenAsync(CancellationToken taskCancellationToken);
+        /// <summary>
+        /// Gets an access token to authorize a request.
+        /// Implementations should handle automatic refreshes of the token
+        /// if they are supported.
+        /// The <paramref name="authUri"/> might be required by some credential types
+        /// (e.g. the JWT access token) while other credential types
+        /// migth just ignore it.
+        /// </summary>
+        /// <param name="authUri">The URI the returned token will grant access to.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
```

Comment:
To me it's not confusing at all. CancellationToken is a widely known concept so cancellationToken variable seems pretty clear to me. Actually, it's used throughout the existing code in numerous places. Leaving as is.